repository/elastic/docs/use.md

```diff
@@ -0,0 +1,131 @@
+# Use an Instance
+
+## Overview
+
+This guide shows how to use an instance of Elastic Search deployed using KUDO.
```

Comment:
Is there a service defined for the operator? Would be nice to know how to access ES from the outside, without bashing into a container...

Reply:
There is *some* service, I don't know whether it's possible to use it from the outside, because the [whole operator is broken under current KUDO](https://github.com/kudobuilder/operators/issues/227). Fixing it, testing access via the service and documenting it are *really* outside the scope of this PR. The point is to really get something that we can link to from the `kudo.dev` site.